multiversx_sdk/accounts/ledger_account.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from multiversx_sdk.core.address import Address
+from multiversx_sdk.core.message import Message
+from multiversx_sdk.core.transaction import Transaction
+from multiversx_sdk.core.transaction_computer import TransactionComputer
+from multiversx_sdk.ledger.ledger_app import LedgerApp
+
+logger = logging.getLogger("ledger_account")
+
+
+class LedgerAccount:
+    def __init__(self, address_index: int = 0) -> None:
+        self.address_index = address_index
+        self.address = self._get_address()
+        self.nonce = 0
+        self.use_hash_signing = True
+
+    def set_address(self, address_index: int = 0):
+        """Sets the working address for the device. Also sets the `address_index` and `address` properties of the class."""
+        app = LedgerApp()
```

Comment:
Just an idea - maybe use lazy initialization pattern for a new (private) member, _self.ledger_app_?

Reply:
as decided, we'll leave it as it is.